### PR TITLE
Update changelog for 1.0.0

### PR DESCRIPTION
**Pull request by WayneDroid**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nidaqmx-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Update CHANGELOG.md for 1.0.0. Once this is complete, contents under 1.0.0 will be updated in the Release.

### Why should this Pull Request be merged?

For 1.0.0 release.

### What testing has been done?

N/A

### CODE REVIEW

**Review by zhindes — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,13 +20,43 @@
...
-    *  ...
+    * Add support for strain calibration (#468 #469)
+    * Add support for DAQmx calibration info properties (#473)
+    * Add support for WaitForValidTimestamp to nidaqmx-python (#479)
+    * Fix raw-read Function Names Spacing Issue (#492)
+    * Update libtime (#504)
+    * Update AI Bridge Default Argument Values (#509)
+    * Improve error reporting in some cases (#550)
+    * Automate Driver Install Experience within nidaqmx Module (#512 #582)
+    * Rewrote examples (#566 #575 #573 #581 #583 #587 #570 #589 #592 #594)
```
Comment (bkeryan) [inline CHANGELOG.md:43]: This list is not in "Title Case", so don't capitalize common nouns.
```suggestion
    * Fix raw-read function names spacing issue (#492)
    * Update libtime (#504)
    * Update AI bridge default argument values (#509)
    * Improve error reporting in some cases (#550)
    * Automate driver install Experience within nidaqmx module (#512 #582)
    * Rewrote examples (#566 #575 #573 #581 #583 #587 #570 #589 #592 #594)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,13 +20,43 @@
...
+
 * ### Major Changes
     * Add support for strain calibration (offset nulling and shunt calibration)
+    * Add support for DAQmx calibration info properties
+    * Add support for WaitForValidTimestamp
     * Fix naming issues:
         * Rename `ShuntCalSelect.AAND_B` to `A_AND_B`.
+    * Automate Driver Install Experience within nidaqmx Module
+    * Update libtime to accept time before 1970
+
```
Comment (bkeryan) [inline CHANGELOG.md:59]: I don't think we should duplicate these items between "merged pull requests" and "major changes", and I think there are too many merged pull requests to list them all out individually.

For "merged pull requests", how about linking to https://github.com/ni/nidaqmx-python/compare/0.9.0...1.0.0 ?

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (WayneDroid) [reply CHANGELOG.md:51]: Agree, too many merged PRs to list out. It's updated as per suggestion.

**Review by bkeryan — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (WayneDroid) [reply CHANGELOG.md]: There are updated as per suggestions.

**Pull request merged**